mobify/progressive-web-scaffold
# Use getasseturl from sdk

Upgrade to version `0.3.0` of the Progressive Web SDK.

## Changes
- Add a `Logo` component to test the `getAssetUrl` function.
- Remove the `cacheHashManifest` webpack alias.
- Use the new `initCacheManifest` function to initialize the cache manifest at app startup.

## How to test-drive this PR
- Make sure `npm run test:all` passes
- Start the dev server with `npm run dev` and preview Merlin's Potions
